src/templates/PoolV2/Overview/index.tsx

```diff
@@ -154,18 +256,89 @@ const Overview = ({ pool, handleClickStakeButton }: IOverviewProps) => {
       <ChartProducts poolId={pool?.id ?? ''} />
 
       <S.ChangeAndStakeContainer>
-        <PriceChange changePriceList={Object.values(changePriceList)} />
-
-        {pool?.chain_id && pool?.pool_id && (
-          <StakeAndEarnCard
-            handleClickStakeButton={handleClickStakeButton}
-            poolName={pool.name}
-            poolIcon={pool.logo ?? ''}
-            poolId={pool?.pool_id ?? undefined}
-            chainId={pool?.chain_id ?? 0}
-            poolPrice={pool?.price_usd ?? '0'}
-          />
-        )}
+        <S.TokenInfoContainer>
+          <div>
+            <Chart
+              data={allocationsDataChart}
+              activeIndex={activeIndex}
+              setActiveIndex={setActiveIndex}
+              isRebalancing={!!RebalancingProgress}
+            />
+          </div>
+
+          <S.TokenInfoContent>
+            <S.ImgAndSymbolWrapper>
+              <img
+                src={tokenSeleted?.token?.logo ?? ''}
+                alt=""
```

Comment:
its a generic alt but helps with something at least
```suggestion
                alt="Token Logo"
```
or maybe add the token name